spec/models/draft_question_spec.rb

```diff
@@ -149,4 +149,39 @@
       end
     end
   end
+
+  describe "answer_settings" do
+    let(:draft_question) { create :draft_question, answer_settings: }
+    let(:answer_settings) do
+      {
+        "hello" => "I have a string as a key",
+        "nested_attributes" => {
+          "name" => "Joe Bloggs",
+        },
+      }
+    end
+
+    it "returns a hash with all keys as symbols" do
+      expect(draft_question.answer_settings).to eq(hello: "I have a string as a key",
+                                                   nested_attributes: {
+                                                     name: "Joe Bloggs",
+                                                   })
+    end
+
+    context "when answer_settings is empty JSON" do
+      let(:answer_settings) { {} }
+
+      it "returns empty hash" do
+        expect(draft_question.answer_settings).to be_empty
+      end
+    end
+
+    context "when answer_settings is nil" do
+      let(:answer_settings) { nil }
+
+      it "returns empty hash" do
+        expect(draft_question.answer_settings).to be_nil
```

Comment:
this `it` block and the one above it both have the text "returns empty hash". This one uses `be_nil,` not `be_empty`. Should the text be different?

A question outside of this PR: does `answer_settings` need a difference between an empty hash and nil?

Some people seem to use active record serializers to do the conversion [like here](https://github.com/rails/rails/issues/25894#issuecomment-777516944). They often replace nil with `{}` for convenience. I don't know if that would help us?

Reply:
@thomasiles Your question about empty hash/nil, I did think about it initially and decided to just return whatever it was set to......Maybe we should return an empty hash. Its a case of changing this one line to return empty hash instead of `raw_setting`.

https://github.com/alphagov/forms-admin/pull/786/files#diff-baf1f2aaaa9c2156d3a608cfd7e96f8f7b85deed52347177f551d1c24f154dc2R14

Reply:
@thomasiles I've pushed a new commit which should default `answer_settings` to empty hash and also resolves the copy issue you spotted in the specs